### PR TITLE
fix: remove unused style

### PR DESCRIPTION
**Pull request by solracsf**
Fix https://github.com/nextcloud/contacts/issues/4148

### CODE REVIEW

**Comment by SebastianKrupinski** [comment]
Tested Works.

Before:
Screenshot 2024-12-05 141310

After:
image

**Review by SebastianKrupinski — APPROVED** [review]
Tested. Works. Thanks for fixing this.

Also thank you for updating the constructor!

**Comment by solracsf** [comment]
Thanks. I'm not familiar with backports policy in this repo but please consider backporting it 👍🏻

**Comment by SebastianKrupinski** [comment]
We normally try to back port fixes 3 versions for all repos, for the contacts app that would be 6.1, 6.0, 5.5.

The only issue is that with the constructor changes, back porting this fix is not possible, version 6.0 supports a min php version of 7.4 which does not support the constructor variable declarations. Also the removal of the addInitScript check makes pack porting to 6.0 impossible as 6.0 supports a min Nc version of 27.

So long story short we can't really back port these changes. Sorry
